admin_manual/enterprise_clients/publishing_android_app.rst

```diff
@@ -0,0 +1,333 @@
+left pull-down activity menu to see the download, tap the app name, and then the 
+installer opens.
+
+.. image:: ../images/android_custom_5.png
+
+.. _publish_google_play:
+
+Publish to the Google Play Store
+--------------------------------
+
+You may elect to publish your app in the Google Play store, either as a free 
+or paid app. There are serveral steps to publishing a free app:
+
+1. Create a Google Play Publisher account.
+
+2. Sign your branded app with your own signing certificate.
+
+3. Upload your signed branded app to your Google Play Publisher account.
+
+As part of creating your Google Play Publisher account you will have to create 
+some screenshots of your app in specific sizes, and create a store description.
+
+Prequisites
+^^^^^^^^^^^
+
+You need three commands to sign your app: ``keytool``, ``jarsigner``, and 
+``zipalign``. ``keytool`` and ``jarsigner`` are in Java runtimes. Linux users 
+can get these in OpenJDK. For example, on current versions of Debian, 
+Mint, and Ubuntu Linux you need to install two packages. The first one supplies 
+``keytool`` and the second one supplies ``jarsigner``::
+
+ $ sudo apt-get install openjdk-7-jre-headless
+ $ sudo apt-get install openjdk-7-jdk
+ 
+It is simpler to get these on CentOS and Fedora, as they have created some nice 
+wrapper scripts around ``keytool`` and ``jarsigner`` that you can install 
+standalone::
+ 
+ $ sudo yum install keytool-maven-plugin.noarch
+ $ sudo yum install maven-jarsigner-plugin.noarch
+ 
+``zipalign`` is included in the `Android Software Development Kit 
+<https://developer.android.com/sdk/index.html>`_. It is a large download, but 
+once you have downloaded it you can copy the ``zipalign`` binary to any 
+computer and use it. Go to `Android Software Development Kit 
+<https://developer.android.com/sdk/index.html>`_ and click the 
+"Download Android Studio" button.
+
+.. image:: ../images/android_custom_17.png
+
+Download the appropriate **SDK Tools Only** package for your operating system.
+
+.. image:: ../images/android_custom_18.png
+
+Unpack it and change to the unpacked directory, for example 
+``$ cd android-sdk-linux`` on Linux systems. There is one more step, and that 
+is to install additional tools::
+ 
+ $ tools/android update sdk --no-ui
+ 
+This will take some time. When it's finished you'll find ``zipalign`` in the 
+``build-tools`` directory. For convenience, you could copy ``zipalign`` to your 
+home folder or other location of your choice, and to any other computer without 
+installing the whole Android SDK.
+
+Digitally Signing Your App
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Google requires that your apps be digitally signed with your own signing 
+certificate, to verify authorship and the authenticity of your app. Self-signed 
+certificates are acceptable. In these examples the name of the app, as supplied 
+by ownBuilder, is ``acmecloud_1.7.0-release-unsigned.apk``.
+
```

Comment:
I´d create the "Create certificate" as a new section because, this is only needed to be done the first time (in case you don't have any certificate.
Next times (whenever an update is release) they won´t have to create the certificate but use the same one. In the case of the Google play, this is important, they have to use the same certificate